libs/langchain/langchain/utilities/arxiv.py

```diff
@@ -141,7 +143,7 @@ def load(self, query: str) -> List[Document]:
                 doc_file_name: str = result.download_pdf()
                 with fitz.open(doc_file_name) as doc_file:
                     text: str = "".join(page.get_text() for page in doc_file)
-            except FileNotFoundError as f_ex:
+            except (FileNotFoundError, fitz.fitz.FileDataError) as f_ex:
```

Comment:
nit: `f_ex`  -> `e` or `exc` is more common, here after the  modification the variabla name `f_ex` doesn't generalize anymore